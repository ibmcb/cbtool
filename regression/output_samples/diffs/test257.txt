 status: vm_24 (cloud-assigned uuid 94FC97FB-7F00-5AAA-88D5-58E80810DBAE) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
 status: Sending a termination request for vm_24 (cloud-assigned uuid 94FC97FB-7F00-5AAA-88D5-58E80810DBAE)....
VM object AFBDA32A-3398-5549-9EA9-D872BA7D7FFE (named "vm_24") could not be attached to this experiment: vm_24 (cloud-assigned uuid 94FC97FB-7F00-5AAA-88D5-58E80810DBAE) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
|Used Memory                                          |39.73M                        
|Uptime (in seconds)                                  |13565158                      
|Total Connections Received                           |6494895                       
|Total Commands Processed                             |93794418                      
|Storage Size                                         |16112459760.0                 
|Data Size                                            |13823017360.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.64046057                 
|Collections                                          |60                            
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|0                             
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|0                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|0                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|0                             
