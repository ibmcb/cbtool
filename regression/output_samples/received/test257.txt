 status: vm_24 (cloud-assigned uuid 94FC97FB-7F00-5AAA-88D5-58E80810DBAE) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
 status: Sending a termination request for vm_24 (cloud-assigned uuid 94FC97FB-7F00-5AAA-88D5-58E80810DBAE)....
VM object AFBDA32A-3398-5549-9EA9-D872BA7D7FFE (named "vm_24") could not be attached to this experiment: vm_24 (cloud-assigned uuid 94FC97FB-7F00-5AAA-88D5-58E80810DBAE) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |39.73M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13565158                      
|Total Connections Received                           |6494895                       
|Total Commands Processed                             |93794418                      
|Number of Keys                                       |331                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16112459760.0                 
|Data Size                                            |13823017360.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.64046057                 
|Collections                                          |60                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |2                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |134                           

 ---------------- VMC-WIDE COUNTERS ----------------
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|0                             
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|0                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|0                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|0                             

