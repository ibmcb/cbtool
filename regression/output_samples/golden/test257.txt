 status: vm_24 (cloud-assigned uuid 46E86429-8AF3-5CAA-BCD5-2450B8AF8C5E) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
 status: Sending a termination request for vm_24 (cloud-assigned uuid 46E86429-8AF3-5CAA-BCD5-2450B8AF8C5E)....
VM object 87292C3E-2517-5E27-B9F1-DFFEC2D53BA6 (named "vm_24") could not be attached to this experiment: vm_24 (cloud-assigned uuid 46E86429-8AF3-5CAA-BCD5-2450B8AF8C5E) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |39.65M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11740400                      
|Total Connections Received                           |6492711                       
|Total Commands Processed                             |92729243                      
|Number of Keys                                       |331                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13966344192.0                 
|Data Size                                            |12133756892.0                 
|Index Size                                           |502881232                     
|Average Object Size                                  |6346.12707571                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |2                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |134                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CB27A579-597C-52AC-B148-644E81A7D93C (simzone_d) VM RESERVATIONS|0                             
|D7D3EB8C-8813-524D-9C2D-7FD7626D4BCC (simzone_b) VM RESERVATIONS|0                             
|97FF48D4-05C4-54D7-B1D8-828EED2A2B20 (simzone_a) VM RESERVATIONS|0                             
|A708E2EA-221A-57ED-BAE8-15077C3E391A (simzone_c) VM RESERVATIONS|0                             

